packages/components/organisms/f-header/src/assets/scss/common.scss

```diff
@@ -18,6 +18,7 @@ $header-buttonIcon-color            : $color-interactive-primary;
 $header-buttonCount-bg              : $color-interactive-primary;
 $header-buttonCount-color           : $color-content-interactive-primary;
 $header-buttonCount-borderColor     : $color-white;
+$header-buttonFocus-borderRadius    : 800px;
```

Comment:
Not sure if this is the same as just using 50% as the border-radius potentially? (usually you can apply 50% as a `border-radius` value to make it rounded as shown in your screenshots)

Reply:
same as above :)